Updated prototype shared-responsibility model.

iMichaela opened the pull request:
# Committer Notes

The prototype shared-responsibility model was updated and the documentation was reviewed. This version of the model is ready for review and integration into the OSCAL v1.2.0 release candidate. The prototype was locally tested with the latest [metaschema-xslt/document-display-deprecation](https://github.com/usnistgov/metaschema-xslt/tree/document-display-deprecation)

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

(For reviewers: [The wiki has guidance](https://github.com/usnistgov/OSCAL/wiki/Issue-Review) on code review and overall issue review for completeness.)

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

david-waltermire commented:
It's hard to see what is changing in this PR, since this is based on an older commit from develop. This is causing a bunch of non-related changes to be reflected. Can this be rebased on the latest from develop to remove the noise?

iMichaela commented:
@david-waltermire - The purpose of the PR and proposed merge into the prototype branch was to be able to generate the documentation: the OSCAL Reference for the updated prototype so the review is easier. The merging onto develop would be next step after the prototype is reviewed. In this case, if something is not favorable, we can address it easier.  Documentation (OSCAL Reference) from the develop branch cannot be created.  

I ran below what you are asking: iMichaela/prototype-shared-responsibility-model  against usnistgov/develop and it is not going to be easier to review also because I had to merge develop into the prototype to test the old prototype with the latest from develop. 
I personally think having the documentation and the outlines generated will be easier. 
I can also generate a features summary using the outline but it will make sense when reviewers have the reference for the prototype updated (there are no major changes from the existing one, more fixing metaschema validation errors and providing the documentation (current OSCAL Reference for the prototypes was not properly documented)
 
```
git log upstream/develop..origin/prototype-shared-responsibility-model  --pretty=oneline --abbrev-commit
```
**Summary:**
d21591ba2 (origin/prototype-shared-responsibility-model) Merge pull request #9 from iMichaela/rc1-shared-responsibility-model
9ad227896 (origin/rc1-shared-responsibility-model, rc1-shared-responsibility-model) Added remarks in provided, responsibility, inherited and satisfied.
054c0b70a Augemnting reference and added implementation-status to component-def per issue 1300.
7a5098c04 Model documentation updates.
06d95ec85 Merge pull request #8 from iMichaela/rc1-shared-responsibility-model
e9131145d Removed implemented-by flag from the responsibility assembly.
a3a2f1651 Merge pull request #7 from iMichaela/rc1-shared-responsibility-model
a1b0ecac6 Fixed xml-to-json conversion
321c42029 Merge pull request #6 from iMichaela/rc1-shared-responsibility-model
6601aacca Cleaned a duplicate definition causing transformation errors.
d0ec68158 Added flag to satisfy to link it to its interited counterpart.
72850142c Minor updates and enhancements.
4bd5bb505 Added responsibility assembly to component def, corrected group-as name and added sr-uuid flag.
45f671297 Made ssp-uuid not required to support SR from non OSCAL SSP and included it also in the leveraged-autorization assembly to support OSCAL SSPs for leveraged systems
9fe6524e1 Re-aligned the SSP with the SR.
b9bc8d129 Added reference documentation and other minor changes
dd6e1a3fa Fixed validation errors in oscal_responsibility-common_metaschema.xml definition file.
5e71f4d62 Merge pull request #5 from iMichaela/develop
af8079bbd (upstream/prototype-shared-responsibility-model, prototype-shared-responsibility-model) Correct errors in metaschema for publishing reference.
:...skipping...
d21591ba2 (origin/prototype-shared-responsibility-model) Merge pull request #9 from iMichaela/rc1-shared-responsibility-model
9ad227896 (origin/rc1-shared-responsibility-model, rc1-shared-responsibility-model) Added remarks in provided, responsibility, inherited and satisfied.
054c0b70a Augmenting reference and added implementation-status to component-def per issue #1300.
7a5098c04 Model documentation updates.
06d95ec85 Merge pull request #8 from iMichaela/rc1-shared-responsibility-model
e9131145d Removed implemented-by flag from the responsibility assembly.
a3a2f1651 Merge pull request #7 from iMichaela/rc1-shared-responsibility-model
a1b0ecac6 Fixed xml-to-json conversion
321c42029 Merge pull request #6 from iMichaela/rc1-shared-responsibility-model
6601aacca Cleaned a duplicate definition causing transformation errors.
d0ec68158 Added flag to satisfy to link it to its interited counterpart.
72850142c Minor updates and enhancements.
4bd5bb505 Added responsibility assembly to component def, corrected group-as name and added sr-uuid flag.
45f671297 Made ssp-uuid not required to support SR from non OSCAL SSP and included it also in the leveraged-autorization assembly to support OSCAL SSPs for leveraged systems
9fe6524e1 Re-aligned the SSP with the SR.
b9bc8d129 Added reference documentation and other minor changes
dd6e1a3fa Fixed validation errors in oscal_responsibility-common_metaschema.xml definition file.
5e71f4d62 Merge pull request #5 from iMichaela/develop
af8079bbd (upstream/prototype-shared-responsibility-model, prototype-shared-responsibility-model) Correct errors in metaschema for publishing reference.
1bec88eb8 Update the snapshot of artifacts.
81ebeaf57 Add missing imports.
c746a974f Updated snapshot for review.
b03cc538e Fix invalid metaschema.
577b97412 Update snapshot.
b9312fd92 Update SR metaschema.
13705b952 Sample snapshot.
:...skipping...
d21591ba2 (origin/prototype-shared-responsibility-model) Merge pull request #9 from iMichaela/rc1-shared-responsibility-model
9ad227896 (origin/rc1-shared-responsibility-model, rc1-shared-responsibility-model) Added remarks in provided, responsibility, inherited and satisfied.
054c0b70a Augemnting reference and added implementation-status to component-def per issue 1300.
7a5098c04 Model documentation updates.
06d95ec85 Merge pull request #8 from iMichaela/rc1-shared-responsibility-model
e9131145d Removed implemented-by flag from the responsibility assembly.
a3a2f1651 Merge pull request #7 from iMichaela/rc1-shared-responsibility-model
a1b0ecac6 Fixed xml-to-json conversion
321c42029 Merge pull request #6 from iMichaela/rc1-shared-responsibility-model
6601aacca Cleaned a duplicate definition causing transformation errors.
d0ec68158 Added flag to satisfy to link it to its interited counterpart.
72850142c Minor updates and enhancements.
4bd5bb505 Added responsibility assembly to component def, corrected group-as name and added sr-uuid flag.
45f671297 Made ssp-uuid not required to support SR from non OSCAL SSP and included it also in the leveraged-autorization assembly to support OSCAL SSPs for leveraged systems
9fe6524e1 Re-aligned the SSP with the SR.
b9bc8d129 Added reference documentation and other minor changes
dd6e1a3fa Fixed validation errors in oscal_responsibility-common_metaschema.xml definition file.
5e71f4d62 Merge pull request #5 from iMichaela/develop
af8079bbd (upstream/prototype-shared-responsibility-model, prototype-shared-responsibility-model) Correct errors in metaschema for publishing reference.
1bec88eb8 Update the snapshot of artifacts.
81ebeaf57 Add missing imports.
c746a974f Updated snapshot for review.
b03cc538e Fix invalid metaschema.
577b97412 Update snapshot.
b9312fd92 Update SR metaschema.
13705b952 Sample snapshot.
82495bcbb Update assemblies - some errors and omissions.
547e35135 Remove export, implementation-status, shared resp. assemblies.
0a5cde861 Remove capabilities assembly from sr.
52d6e2d1c Remove import-component-definition from sr.
f97d3d783 Remove unexpected inherited and satisfied from export.
:...skipping...
d21591ba2 (origin/prototype-shared-responsibility-model) Merge pull request #9 from iMichaela/rc1-shared-responsibility-model
9ad227896 (origin/rc1-shared-responsibility-model, rc1-shared-responsibility-model) Added remarks in provided, responsibility, inherited and satisfied.
054c0b70a Augemnting reference and added implementation-status to component-def per issue 1300.
7a5098c04 Model documentation updates.
06d95ec85 Merge pull request #8 from iMichaela/rc1-shared-responsibility-model
e9131145d Removed implemented-by flag from the responsibility assembly.
a3a2f1651 Merge pull request #7 from iMichaela/rc1-shared-responsibility-model
a1b0ecac6 Fixed xml-to-json conversion
321c42029 Merge pull request #6 from iMichaela/rc1-shared-responsibility-model
6601aacca Cleaned a duplicate definition causing transformation errors.
d0ec68158 Added flag to satisfy to link it to its interited counterpart.
72850142c Minor updates and enhancements.
4bd5bb505 Added responsibility assembly to component def, corrected group-as name and added sr-uuid flag.
45f671297 Made ssp-uuid not required to support SR from non OSCAL SSP and included it also in the leveraged-autorization assembly to support OSCAL SSPs for leveraged systems
9fe6524e1 Re-aligned the SSP with the SR.
b9bc8d129 Added reference documentation and other minor changes
dd6e1a3fa Fixed validation errors in oscal_responsibility-common_metaschema.xml definition file.
5e71f4d62 Merge pull request #5 from iMichaela/develop
af8079bbd (upstream/prototype-shared-responsibility-model, prototype-shared-responsibility-model) Correct errors in metaschema for publishing reference.
1bec88eb8 Update the snapshot of artifacts.
81ebeaf57 Add missing imports.
c746a974f Updated snapshot for review.
b03cc538e Fix invalid metaschema.
577b97412 Update snapshot.
b9312fd92 Update SR metaschema.
13705b952 Sample snapshot.
82495bcbb Update assemblies - some errors and omissions.
547e35135 Remove export, implementation-status, shared resp. assemblies.
0a5cde861 Remove capabilities assembly from sr.
52d6e2d1c Remove import-component-definition from sr.
f97d3d783 Remove unexpected inherited and satisfied from export.
779b4db1d Commit artifacts for review.
8dbbe731f Add shared responsibility model prototype.
c5713cd5d Merge branch 'prototype-shared-responsibility-model' of https://github.com/usnistgov/OSCAL into prototype-shared-responsibility-model
0f02760b1 Add satisfied uuid
098e0e5a4 Deprecate export
54f539ce0 Correct naming
bd76aebe0 Adjust assemblies
662685486 Looking at a few options.
e7e7591f9 Fix shared flag.
1222c087f Adjust wording.
8941eb321 Try exportable attribute on shared-responsibility.
39ab37691 correct exportable tag.
5a03de78f Add exportable attribute to implementation-status
cae8a0b73 Update assemblies.
ae65ea3ce Separate assemblies for  provided and responsibility
e056da621 Add inherited and satisfied
f29d09ce4 Place shared assemblies and flags in a common
8eef8b57e Initial add of shared-responsibility assembly.
8b8857f5d Add satisfied uuid
4722eeb02 Deprecate export
f6f63b476 Correct naming
070db28bd Adjust assemblies
8302c2c3b Looking at a few options.
aba6cc8ed Fix shared flag.
b578dd997 Adjust wording.
6f9386f45 Try exportable attribute on shared-responsibility.
a5706f86a correct exportable tag.
44d3f2c7b Add exportable attribute to implementation-status
d9c21f09f Update assemblies.
ff0047afe Separate assemblies for  provided and responsibility
dbfe09524 Add inherited and satisfied
2d0e5c558 Place shared assemblies and flags in a common
9edc8138b Initial add of shared-responsibility assembly.
e5b6e790b Bump markdown-link-check from 3.10.2 to 3.11.2 in /build (#1864)
948e88ad0 Bump actions/checkout from 3.3.0 to 3.5.3 (#1863)
2a908809d Bump actions/setup-node from 3.6.0 to 3.7.0 (#1854)
d6aed1c19 Add XSLT runner utility (#1879)
64e130fb2 Script and dependency fixes for v1.1.0 content updates (#1874)
05ee674eb Remove dead code for biblio for #1841 (#1875)
e9cce42b5 (upstream/define-5-common-responsibility-assemblies) [skip ci] Address PR feedback from @iMichaela.
6374f4100 [skip ci] Clarify guidance for usnistgov/OSCAL#1864. (#1869)
1f45b5259 Added artifact usage + corrected dead links (#1865)
